Dockerfile

```diff
@@ -24,20 +24,20 @@ RUN tar xzf epl.tgz --strip-components=1
 
 ENV NODE_ENV=production
 RUN cd src && \
-      npm install
+      npm install --no-progress --no-audit
 
 # Install extra plugins
-COPY package.json package.json
+COPY package.json package-lock.json /builder/
 COPY src/plugins /builder/src/plugins/
-RUN npm install
+RUN npm install --no-progress --no-audit
```

Comment:
`--no-progress` is still a performance improvement ?

Reply:
No idea, but this prevent breaking the build logs.